src/Sulu/Component/Security/Authorization/SecurityChecker.php

```diff
@@ -39,9 +39,8 @@ public function __construct(
 
     public function hasPermission($subject, $permission)
     {
-        if (!$subject || !$this->tokenStorage->getToken()) {
+        if (!$subject) {
             // if there is no subject the operation is allowed, since we have nothing to check against
-            // if there is no token we are not behind a firewall, so the action is also allowed (e.g. command execution)
```

Comment:
we decided that this is not the correct place to disable security checks during command execution. instead of disabling checks automatically, out services should allow to turn off check explicitly like the `MediaManager::delete()` method:
https://github.com/sulu/sulu/blob/b5e6afab84ea8c01d08be5890aa21c0d244a331b/src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManagerInterface.php#L89